website/versioned_docs/version-0.60/_integration-with-existing-apps-objc.md

```diff
@@ -280,7 +280,7 @@ First `import` the `RCTRootView` header.
 }
 ```
 
-> Note that `RCTRootView initWithURL` starts up a new JSC VM. To save resources and simplify the communication between RN views in different parts of your native app, you can have multiple views powered by React Native that are associated with a single JS runtime. To do that, instead of using `[RCTRootView alloc] initWithURL`, use [`RCTBridge initWithBundleURL`](https://github.com/facebook/react-native/blob/master/React/Base/RCTBridge.h#L93) to create a bridge and then use `RCTRootView initWithBridge`.
+> Note that `RCTRootView initWithURL` starts up a new JSC VM. To save resources and simplify the communication between RN views in different parts of your native app, you can have multiple views powered by React Native that are associated with a single JS runtime. To do that, instead of using `[RCTRootView alloc] initWithURL`, use [`RCTBridge initWithBundleURL`](https://github.com/facebook/react-native/blob/main/packages/react-native/React/Base/RCTBridge.h#L93) to create a bridge and then use `RCTRootView initWithBridge`.
```

Comment:
For all of these versioned documents, maybe we should keep the existing organization, but point to e.g. `0.60-stable` instead of master/main?

Reply:
yeah that makes the most sense, but it would be a lot harder and require a bit manual intervention. I don't think I would have the time to do that right now unfortunately, but what I could do is at least get the current docs pointed at the right place and save the older ones for later?